src/tools/wasm-split/instrumenter.cpp

```diff
@@ -102,21 +125,27 @@ void Instrumenter::instrumentFuncs() {
       });
       break;
     }
-    case WasmSplitOptions::StorageKind::InMemory: {
+    case WasmSplitOptions::StorageKind::InMemory:
+    case WasmSplitOptions::StorageKind::InSecondaryMemory: {
       if (!wasm->features.hasAtomics()) {
-        Fatal() << "error: --in-memory requires atomics to be enabled";
+        Fatal() << "error: --in-memory and --in-secondary-memory requires "
```

Comment:
```suggestion
        Fatal() << "error: --in-memory and --in-secondary-memory require "
```

Or alternatively we could specialize the error message to use only the flag the user passed, which would be nicer IMO.

Reply:
Specializing is much nicer 👍🏾